from app.domain.data_models.common import ResponseError

class ResponseInternalError(ResponseError):
    pass
